HousingRepairsOnlineApi/UseCases/ChangeRepairRequestUseCase.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Threading.Tasks;
+using Ardalis.GuardClauses;
+using HousingRepairsOnlineApi.Domain;
+using HousingRepairsOnlineApi.Gateways;
+using HousingRepairsOnlineApi.Helpers;
+
+namespace HousingRepairsOnlineApi.UseCases
+{
+    public class ChangeRepairRequestUseCase : IChangeRepairRequestUseCase
+    {
+        private readonly IRepairStorageGateway repairStorageGateway;
+
+        public ChangeRepairRequestUseCase(IRepairStorageGateway repairStorageGateway)
+        {
+            this.repairStorageGateway = repairStorageGateway;
+        }
+
+        public async Task Execute(Repair repair, DateTime startDateTime, DateTime endDateTime)
+        {
+            Guard.Against.Null(repair, nameof(repair));
+            repair.Time.StartDateTime = startDateTime;
+            repair.Time.EndDateTime = endDateTime;
```

Comment:
This is setting the `StartDateTime` and `EndDateTime` but is leaving the `Display` value as it is.
Anywhere the `Display` value is used won't correctly reflect the changed appointment.
Known locations are:
- Frontend page showing the repair details
- In notification sent to resident